deployer/deployer.js

```diff
@@ -80,8 +81,12 @@ class Deployer {
 			throw new Error(`Passed contract is not a valid contract object. It needs to have bytecode, abi and contractName properties`);
 		}
 
+		if (!isValidBytecode(contract.bytecode)) {
+			throw new Error(`The bytecode of the contract doesn't contain а library`);
```

Comment:
I do not think this error message is correct as this checks if bytecode is string more then 1 symbol long. Lets change this message :)